slices/ovn-central.yaml

```diff
@@ -0,0 +1,10 @@
+package: ovn-central
+
+slices:
```

Comment:
Should there be a `libs` (or `services`) slice as well? Aren't the following needed?
```
/lib/systemd/system/ovn-central.service
/lib/systemd/system/ovn-northd.service
/lib/systemd/system/ovn-ovsdb-server-nb.service
/lib/systemd/system/ovn-ovsdb-server-sb.service
```

Reply:
These are all systemd related service definitions which I don't think are relevant for ROCK's